stats/opencensus/opencensus.go

```diff
@@ -152,13 +152,24 @@ func setRPCInfo(ctx context.Context, ri *rpcInfo) context.Context {
 	return context.WithValue(ctx, rpcInfoKey{}, ri)
 }
 
-// getSpanWithMsgCount returns the rpcInfo stored in the context, or nil
+// getRPCInfo returns the rpcInfo stored in the context, or nil
 // if there isn't one.
 func getRPCInfo(ctx context.Context) *rpcInfo {
 	ri, _ := ctx.Value(rpcInfoKey{}).(*rpcInfo)
 	return ri
 }
 
+// GetTraceAndSpanID returns the trace and span ID of the span in the context.
+// Returns true if IDs present and false if IDs not present.
+func GetTraceAndSpanID(ctx context.Context) (trace.TraceID, trace.SpanID, bool) {
+	ri, ok := ctx.Value(rpcInfoKey{}).(*rpcInfo)
+	if !ok {
+		return [16]byte{}, [8]byte{}, false
```

Comment:
I think(?) this will work instead and be a little cleaner: `return trace.TraceID{}, trace.SpanID{}, false`.

Reply:
Switched.